index.html

```diff
@@ -19,19 +20,26 @@
           <div class="logo">
             <a id="anc" href="#">Debby Miressa</a>
           </div>
-          <div class="space"></div>
+          <div class="space">
+            <a class="port" href="#">Portfolio</a>
+            <a class="bout" href="#">About</a>
+            <a class="cont" href="#">Contact</a>
+            <a class="mail"><img src="images/Mail.png" alt="mail"></a>
+          </div>
           <button id="menu" class="button">
             <img src="images/Menu@1x.png" alt="Header" />
           </button>
         </header>
 
         <section class="headline">
+          <div class="main-headline">
```

Comment:
The icons are not aligned as per the Figma template. Kindly fix it.
| Your design  | Figma design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/33268927/177905559-f8a19452-e0bf-460d-8768-7562d1944d7a.png) | ![image](https://user-images.githubusercontent.com/33268927/177905485-7352f6d3-8e12-46f6-8876-379170d7afad.png) |